src/stacks_events.rs

```diff
@@ -0,0 +1,86 @@
+use blockstack_lib::chainstate::stacks::StacksTransaction;
+use chrono::{DateTime, Local, SecondsFormat, Utc};
+use lazy_static::lazy_static;
+use regex::Regex;
+use serde_json::{json, Value};
+use stacks_common::codec::StacksMessageCodec;
+use stacks_common::util::hash::hex_bytes;
+use std::fs::File;
+use std::io::prelude::*;
+use std::io::BufReader;
+use std::net::TcpListener;
+use std::net::TcpStream;
+use std::time::SystemTime;
+use std::{env, io};
+
+fn main() {
+    serve_for_events();
+}
+
+fn serve_for_events() {
+    let addr = "127.0.0.1:3700";
```

Comment:
Should this port be configurable, or at least documented?

Reply:
I'll add a command argument parser.